src/content/terraform/docs-landing.json

```diff
@@ -1,110 +1,82 @@
-					"title": "Transit Secrets Engine",
-					"url": "/vault/docs/secrets/transit"
+					"description": "Describe infrastructure on various providers with Terraform's configuration language.",
+					"title": "Configuration Language",
+					"url": "/terraform/language"
 				},
 				{
-					"description": "Handle secure data transformation and tokenization against a provided input value using NIST vetted cryptographic standards and other transformations such as masking.",
-					"title": "Transform Secrets Engine",
-					"url": "/vault/docs/secrets/transform"
+					"description": "Use the Terraform CLI to manage configuration, plugins, infrastructure, and state.",
+					"title": "Transform CLI",
+					"url": "/terraform/cli"
 				},
 				{
-					"description": "Exchange a sensitive value for an unrelated value called a token. The original sensitive value cannot be recovered from a token alone but is irreversible.",
-					"title": "Tokenization",
-					"url": "/vault/docs/secrets/transform/tokenization"
+					"description": "Use CDK for Terraform to write and deploy Terraform configurations in your preferred programming language.",
+					"title": "CDK for Terraform",
+					"url": "/terraform/cdktf"
 				}
 			]
 		},
 		{
 			"type": "card-grid",
-			"title": "Key Management",
-			"description": "Use a standardized workflow for distribution and lifecycle management of cryptographic keys in various KMS providers.",
+			"title": "Develop and Share",
 			"cards": [
 				{
-					"description": "Act as a key management interoperability protocol server provider and handle the lifecycle of its KMIP managed objects.",
-					"title": "KMIP Secrets Engine",
-					"url": "/vault/docs/secrets/kmip"
+					"description": "Terraform Cloud helps teams use Terraform together, with version control, state sharing, governance, and more.",
+					"title": "Terraform Cloud",
+					"url": "/terraform/cloud-docs"
 				},
 				{
-					"description": "A consistent workflow for distribution and lifecycle management of cryptographic keys in key management service (KMS) providers.",
-					"title": "Key Management Secrets Engine",
-					"url": "/vault/docs/secrets/key-management"
-				},
-				{
-					"description": "Generate dynamic X.509 certificates without going through the manual process of generating a private key and CSR or submitting to a CA.",
-					"title": "PKI",
-					"url": "/vault/docs/secrets/pki"
+					"description": "Terraform Enterprise is a self-hosted instance of Terraform Cloud, which is ideal for organizations that have strict security and compliance requirements.",
+					"title": "Terraform Enterprise",
+					"url": "/terraform/enterprise"
 				}
 			]
 		},
 		{
-			"type": "section-heading",
-			"title": "Developers"
-		},
-		{
-			"type": "icon-card-grid",
+			"type": "card-grid",
+			"title": "Collaborate",
```

Comment:
I think this should be "Develop and Share"?

Reply:
yeah .....i don't know what happened here - thank you for catching this!